### ISSUE TITLE
Add guidance for fragment processing

### ISSUE THREAD

**Issue by msporny** [ietf-wg-mediaman/suffixes#3]
On Tue, Oct 18, 2022 at 5:28 AM Roberto Polli wrote:
> On mediaman-suffixes, at a first read of mediaman-suffixes I think that the "generic processing" described in https://www.ietf.org/archive/id/draft-ietf-mediaman-suffixes-02.html#name-processing-multiple-suffixe  can be tricky with regards to fragment identification (see the long discussion we had on that topic) but this is also true for other media types: it is thus something that  should be analyzed in mediaman-suffixes.

**Comment by msporny** [comment]
It is true that one cannot assume the same semantic meaning for a fragment between various different structured syntax suffixes. 

We might want to add guidance noting that an implementer will have to ensure that the semantics are the same in their processing pipelines if multiple structured syntax suffixes are processed in a multi-step fashion.

**Comment by msporny** [comment]
From IETF 115 in London: We need to add text to the spec that states that if you want to do fragment processing that those rules should be associated with the specification associated with the structured syntax suffix.

**Comment by msporny** [comment]
From Darrel Miller: 
> It appears that in the suffix registry it explicitly says what the fragment should be https://www.iana.org/assignments/media-type-structured-suffix/media-type-structured-suffix.xhtml e.g. for +zip it says that the fragment should be for zip.

**Comment by msporny** [comment]
Added language in 6bf7646c478ff93ca2e448bb27a6f3fe55814884.

It turns out that a lot of the fragment processing rules were copy-pasted between entries in the "Fragment Identifier Considerations" of the structured syntax suffixes registry: https://www.iana.org/assignments/media-type-structured-suffix/media-type-structured-suffix.xhtml

I lifted that language and put it into the specification so we can remove all the repetitive copy-paste in the registry while allowing us to preserve the non-copy-pasted content in the SSS registry.

I'd appreciate a review on the commit above, @alvestrand, given that this issue overlaps a bit with #6. 

I'm marking this issue as resolved now that there is clear guidance for fragment processing for structured syntax suffixes.
